gemrb/core/Interface.cpp

```diff
@@ -2180,15 +2180,14 @@ bool Interface::InitializeVarsWithINI(const path_t& iniFileName)
 		overrides = defaults;
 	}
 
-	for (int i = 0; i < defaults->GetTagsCount(); i++) {
-		const StringView tag = defaults->GetTagNameByIndex(i);
-		for (int j = 0; j < defaults->GetKeysCount(tag); j++) {
-			auto key = std::string{defaults->GetKeyNameByIndex(tag, j).c_str()};
-			//skip any existing entries. GemRB.cfg has priority
+	for (auto& group : *defaults) {
+		for (auto& pair : group) {
+			// skip any existing entries. GemRB.cfg has priority
+			const auto& key = pair.first;
 			auto lookup = vars.Get(key);
 			if (lookup == nullptr) {
-				ieDword defaultVal = defaults->GetKeyAsInt(tag, key, 0);
-				vars.Set(key, overrides->GetKeyAsInt(tag, key, defaultVal));
+				auto value = defaults->GetKeyAsInt(group.GetName(), key, 0);
```

Comment:
Maybe simpler as `auto value = atoi(pair.second);`?

Reply:
Yeah, just goes around the interface, will think again of this.

Reply:
Found something better.